ceph-object-corpus

```diff
@@ -1 +1 @@
-Subproject commit 7bd9bbe235c59527dd1ab192fed21944da686d20
+Subproject commit da43eb119558fb46d86e62f3b82d5ca63546f69e
```

Comment:
thanks to @dachary , we need to remove this commit from this pull request. and it will be good then =D